platforms/fabric/src/main/java/com/dfsek/terra/fabric/mixin/implementations/ConfiguredFeatureMixin.java

```diff
@@ -0,0 +1,42 @@
+package com.dfsek.terra.fabric.mixin.implementations;
+
+import com.dfsek.terra.api.math.vector.Location;
+import com.dfsek.terra.api.platform.world.Tree;
+import com.dfsek.terra.api.util.collections.MaterialSet;
+import com.dfsek.terra.fabric.TerraFabricPlugin;
+import com.dfsek.terra.profiler.ProfileFrame;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.registry.BuiltinRegistries;
+import net.minecraft.world.StructureWorldAccess;
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import org.spongepowered.asm.mixin.Implements;
+import org.spongepowered.asm.mixin.Interface;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+import java.util.Locale;
+import java.util.Random;
+
+@Mixin(ConfiguredFeature.class)
+@Implements(@Interface(iface = Tree.class, prefix = "terra$", remap = Interface.Remap.NONE))
+public abstract class ConfiguredFeatureMixin {
```

Comment:
What does this class do?
If it's a tree, then it should be renamed to that.

Reply:
It's a mixin, mixing into ConfiguredFeature. Therefore it should be called ConfiguredFeatureMixin